pom.xml

```diff
@@ -46,7 +46,7 @@
         <air.maven.version>3.3.9</air.maven.version>
 
         <dep.antlr.version>4.5.1</dep.antlr.version>
-        <dep.airlift.version>0.128</dep.airlift.version>
+        <dep.airlift.version>0.128-SNAPSHOT</dep.airlift.version>
```

Comment:
I'm confused; if 0.128 is on master, that suggests that 0.128 is a released version of airlift, and therefore 0.128-SNAPSHOT wouldn't be changing. Am I missing something?